docs/fastdds/persistence/persistence.rst

```diff
@@ -56,9 +56,17 @@ or DataReader) |PropertyPolicyQos|.
   For selecting an appropriate GUID for the DataReader and DataWriter, please refer to
   `RTPS standard <https://www.omg.org/spec/DDSI-RTPS/2.2/PDF>`_ (section *9.3.1 The Globally Unique Identifier (GUID)*).
 
+  If no ``dds.persistence.guid`` is specified,
+  the durability behavior will fallback to |TRANSIENT_LOCAL_DURABILITY_QOS-api|.
+
```

Comment:
Do we need the not below if we have this text here?

Reply:
I think that it complains about the line too long otherwise

Reply:
No we dont, adding it